vagrant.rb

```diff
@@ -170,7 +170,7 @@
 $SKIP_IP ||= 1
 
 # Configure a complete platform by just providing an id and a json file
-def platform(config, pf_id, pf_name, override={})
+def platform(config, pf_id, pf_name, override={}, upgrade=True)
```

Comment:
I am almost sure it should be False by default, and the Platform object instanciated by the parser when calling the "update-rudder" cli should set the boolean to true